### ISSUE TITLE
loss

### ISSUE THREAD

**Issue by XIAO1HAI** [dahyun-kang/ifsl#5]
Hello, excuse me. I would like to ask if the corresponding loss of weak label and strong label in your paper is Lc and Ls respectively. However, there is an item in the appendix that compares the corresponding results of Lc, Ls and Lc + Ls, how does Lc + Ls implement dene in the code？ Looking forward to hearing from you, Thanks.

**Comment by dahyun-kang** [comment]
Hello!

- if the corresponding loss of weak label and strong label in your paper is Lc and Ls

Yes.

- how does Lc + Ls implement dene in the code

https://github.com/dahyun-kang/ifsl/blob/eb2cf9604ff5d55e3a158d25ebc27fcb32ada067/fs-cs/model/ifsl.py#L57-L60

I simply hardcoded by adding the above two losses with 0.1 multiplied to the weak label loss (Lc).


Have a great day! 😃 

Dahyun.

**Comment by XIAO1HAI** [comment]
Thank you very much for your reply！

Regarding the hardcode you mentioned, can I understand the test indicator of ASNet Lc + Ls (as shown below) in the third line of the article, and directly train it by changing loss to loss = Ls + 0.1 * Lc?
 
![image](https://user-images.githubusercontent.com/90389233/196392788-65535ed0-0f2d-40bc-8191-7a16bd1f9ac3.png)

**Comment by dahyun-kang** [comment]
Yes, exactly.

**Comment by XIAO1HAI** [comment]
Ok, thank you very much for your reply, very helpful！

Hope to have the opportunity to communicate more.

**Comment by dahyun-kang** [comment]
Haha my pleasure, thank you for your attention on the work.
You are more than welcome to chat here or reach out to me.
